doc/access-logs.md

```diff
@@ -1,5 +1,43 @@
 ## Understanding Access Logs
 
+### Fields
+
+* `app_id`: Cloud Foundry application GUID
+* `authority`: Authority (for HTTP/2) or Host header for HTTP/1.1 of the request; see [RFC7540](https://tools.ietf.org/html/rfc7540#section-8.1.2.3) for more details
+* `bytes_received`: Body bytes received, [%BYTES_RECEIVED% in Envoy](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log.html?highlight=BYTES_RECEIVED)
+* `bytes_sent`: Body bytes sent. For WebSocket connection it will also include response header bytes; [%BYTES_SENT% in Envoy](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log.html?highlight=BYTES_SENT)
```

Comment:
```suggestion
* `bytes_sent`: Body bytes sent. For WebSocket connections it will also include response header bytes; [%BYTES_SENT% in Envoy](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log.html?highlight=BYTES_SENT)
```